### PR TITLE
Dersana mikael

### PR DESCRIPTION
**Pull request by eu-genia**
I have partially restored the general DM record to have back all text parts that had disappeared

I have moved texts and relations to the individual homilies

I have updated the general record on Miracles

Here https://github.com/BetaMasaheft/Manuscripts/pull/2742 I have updated the links in the manuscript records

To Do: create individual records for miracles and associate them with DM and MM

Move text bits to individual miracles

Make the general record clean as soon as parts are moved out - will be gradually done

### CODE REVIEW

File: 1001-2000/LIT1295Dersan.xml [hunk 1]
```diff
@@ -152,160 +141,88 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                     <gap reason="omitted" unit="chars" quantity="3"/> ውእቱ፡ እግዚአብሔር፡ ስ<choice><sic>ብ</sic><corr>ቡ</corr></choice>ሕ፡ 
+                     ወጥቀ፡ ልዑል፡ ዲበ፡ <choice><sic>ኩ</sic><corr>ኵ</corr></choice>ሉ፡ ፍጥረት፡ ዝኢያገምሮ፡ ህሊና፡ ሰብእ፡ 
+                     ወኢአምሮ፡ ዘመላእክት፡ ለፈጣሬ፡ ሰማያት፡ ወምድር፡ እስመ፡ ውእቱ፡ አዘዘ፡ በቃለ፡ ጥበቡ፡ ወአቀሞሙ፡ ለዓለም፡ ትእዝዝ፡ ወሀቦሙ፡ ወኢሃለፉ፡ 
+                     <gap reason="omitted" unit="lines" quantity="10"/> ወለእለ፡ ይትለአከዎ፡ ነደ፡ እሳት፡ ስልስ፡ ዲቤ፡ ዝያ፡ ትአቦን፡ መልአከ፡ እግዚአብሔር፡ አውዶሙ፡ ለእለ፡
+                     ይሪፈርህዎ፡ ወያድኅኖሙ፡ ጠአሙ፡ ወታእምሩ፡ ከመ፡ ሄር፡ እግዚአብሔር፡ ወመንከር፡ ጥበብከ፡ እግዚኦ፡ ስብሐት፡ ለብ፡ ስብሐት፡ ለወልድ፡ ስብሐት፡ ለመንፈስ፡ ቅዱስ፡ ይዜኒ፡ 
+                     ወዘልፈኒ፡ ወለአለመ፡ ዓ፡ ዓ፡</ab>
+               </div>
+               
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="CommemorativeJoshua">
+                  <label xml:lang="en">Commemorative notice on Michael's appearance to Joshua</label>
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LIT1295Dersan.xml:204]: The following text is based on the account in `<bibl><ptr target="bm:2006DersanaMikael"/></bibl>`, but not on the manuscript record in BMLacq776.

File: 1001-2000/LIT1295Dersan.xml [hunk 2]
```diff
@@ -432,66 +342,82 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                      </ab>
                   </div>
                </div>
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamAqabe" corresp="LIT6822SalamLakaAqabe">
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamTahsas" corresp="LIT6822SalamLakaAqabi">
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LIT1295Dersan.xml:345]: You kept the reference to the month for this salam, while you changed the IDs for other salams.

File: 1001-2000/LIT1295Dersan.xml [hunk 3]
```diff
@@ -607,87 +499,58 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                </div>
                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamBeauties">
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LIT1295Dersan.xml:500]: Here you kept the new ID 'SalamBeauties' and dismissed the older reference to the month to indicate this salam.

File: 1001-2000/LIT1295Dersan.xml [hunk 4]
```diff
@@ -1443,43 +1140,57 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                  <l n="2">ወይቅጽበነ፡ ፍቁረ፡ አምላክ፡ ለደቂቀ፡ ዛቲ፡ መርጡል፡ ወይቀጥቅጦ፡ በቅድሜነ፡ ለመስቴማ፡ ሀጕል፡ ወያዕድወነ፡ በአክናፊሁ፡ እምባሕረ፡ እሳት፡ ዘሲኦል፡
+                     ወይሰደነ፡ ቤተ፡ ባዕል፡ ውስተ፡ ፒላሳሁ፡ ለዐማኑኤል፡ ወይሴስየነ፡ እምዘገነት፡ አስካል፡ ወያስምዐነ፡ እምዘምሕረት፡ ቃል፡ ለኵልነ፡ ሰማዕያን፡ ለዓለመ፡ ዓለም፡ አሜን።</l>
+               </ab>
+            </div>
+            
+            <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="MiracleImpure">
+               <label>Miracle about one of the impure children of Adam</label>
+               <note>Not attested in any manuscripts.</note>
+               <div type="textpart" subtype="incipit">
                   <ab>
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LIT1295Dersan.xml:1194]: Citet after EOTC edition.

File: 1001-2000/LIT1295Dersan.xml [hunk 5]
```diff
@@ -1492,76 +1203,66 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+                  <l n="2">ድኅረ፡ ኃለፈ፡ ክረምት፡ ወድኅረ፡ ገብአ፡ ዝናም፡ እንተ፡ አስተርአይከ፡ እግዚኦ፡ ጽጌያተ፡ ገዳም፡ አፈወ፡ ሃይማኖት፡ ነአ፡ ልድ፡ ምስለ፡ ፍሬ፡ ምግባር፡
+                     ጥዑም፡ አትግሀነ፡ ለስብሐቲከ፡ ከመ፡ ትጋሆሙ፡ ዮም፡ ለፅብስት፡ ንህብ፡ ወሀካይ፡ ቃሕም።</l>
+               </ab>
             </div>
+               
                <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Prayer">
                   <label>Za-ʾaqrabku māḥleta… </label>
                   <note>Attested in <ref type="mss" corresp="ESnsm006"/>.</note>
-                  <div type="textpart" subtype="incipit">
+                  <div type="textpart" subtype="inplicit">
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LIT1295Dersan.xml:1246]: incipit or implicit?

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: 3001-4000/LIT3960Homily.xml [hunk 6]
```diff
@@ -56,6 +58,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+               <bibl>
+                  <ptr target="bm:Hammerschmidt1988Anthologia"/>
+                  <citedRange unit="page">20-24</citedRange>
+               </bibl> 
+               
+                  <bibl>
+                     <ptr target="bm:Tasfa1954DersanaM"/>
+                     <citedRange unit="page">69-74</citedRange>
+                  </bibl>
+                  <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3960Homily.xml:71]: add: `<citedRange unit="page">103-111</citedRange>`

File: 1001-2000/LIT1295Dersan.xml (hunk 2 above)
Comment (eu-genia) [reply 1001-2000/LIT1295Dersan.xml:345]: To save work I refrained from xhanging Id for texts that do not jump around the year

File: 3001-4000/LIT3961Homily.xml [hunk 7]
```diff
@@ -53,9 +55,27 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             </listRelation>
+            <listBibl type="secondary"><bibl><ptr target="bm:Zotenberg1877Catalogue"/><citedRange unit="page">207a</citedRange></bibl></listBibl>
+            <listBibl type="editions">
+               <bibl><ptr target="bm:budge1894stmichael"/></bibl>
+               
+               <bibl>
+                  <ptr target="bm:Tasfa1954DersanaM"/>
+                  <citedRange unit="page">78-103</citedRange>
+               </bibl>
+               <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3961Homily.xml:76]: add: `<citedRange unit="page">116-157</citedRange>`

File: 3001-4000/LIT3962Homily.xml [hunk 8]
```diff
@@ -53,13 +59,23 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
       <body>
          <div type="bibliography">
+            <listBibl type="editions">
+               <bibl><ptr target="bm:budge1894stmichael"/></bibl>
+               
+               <bibl>
+                  <ptr target="bm:Tasfa1954DersanaM"/>
+                  <citedRange unit="page">107-114</citedRange>
+               </bibl>
+               <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3962Homily.xml:69]: add: `<citedRange unit="page">164-176</citedRange>`

File: 3001-4000/LIT3963Homily.xml [hunk 9]
```diff
@@ -55,9 +60,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             </listRelation>
+            <listBibl type="secondary">
+               <bibl><ptr target="bm:Lusini2005EAEDersanaM"/></bibl>
+            </listBibl>
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Tasfa1954DersanaM"/>
+                  <citedRange unit="page">119-122</citedRange>
+               </bibl>
+               <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3963Homily.xml:81]: add: `<citedRange unit="page">185-190</citedRange>`

File: 3001-4000/LIT3963Homily.xml [hunk 10]
```diff
@@ -34,7 +35,11 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <abstract>
-            <p/>
+            <p>Homily on <persName ref="PRS3899Eudoxia"/>, wife of Emperor <persName ref="PRS2016Arcadius"/> and on 
+               a certain Yoḥannǝs, who could be identified with <persName ref="PRS10390Yohanne"/> 
+               following <bibl><ptr target="bm:Lusini2005EAEDersanaM"/></bibl>, but according to <persName ref="DN"/>, is more 
+               likely to be identified with <persName ref="PRS5720JohnChr"/> due to the manuscripts' usual formulation 
+               <q xml:lang="gez">baʾǝnta Yoḥannǝs</q>.</p>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3963Homily.xml:42]: add: "It is usually transmitted in the section for the month Gǝnbot."

File: 3001-4000/LIT3964Homily.xml [hunk 11]
```diff
@@ -5,16 +5,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       <fileDesc>
          <titleStmt>
             <title xml:id="t1">Homily on the Archangel Michael (for Sane)</title>
+            <title xml:id="t2"> Homily for the 12th day of Sane on  Astaraniqos and Euphemia by Yoḥannǝs</title>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3964Homily.xml:8]: Delete space before homily and one of the two spaces before Astaraniqos

File: 3001-4000/LIT3964Homily.xml [hunk 12]
```diff
@@ -54,8 +58,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
+               <relation name="ecrm:P129_is_about" active="LIT3964Homily" passive="PRS14019Euphemia"></relation>
                <relation name="saws:isAttributedToAuthor" active="LIT3964Homily" passive="PRS10382Yohanne"/>
+               <relation name="saws:isAttributedToAuthor" active="LIT3964Homily" passive="PRS13526Yohannes"></relation>
             </listRelation>
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Tasfa1954DersanaM"/>
+                  <citedRange unit="page">127-136</citedRange>
+               </bibl>
+               <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3964Homily.xml:71]: add: `<citedRange unit="page">198-212</citedRange>`

File: 3001-4000/LIT3965Homily.xml [hunk 13]
```diff
@@ -53,6 +56,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <div type="bibliography">
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Tasfa1954DersanaM"/>
+                  <citedRange unit="page">153-160</citedRange>
+               </bibl>
+               <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3965Homily.xml:64]: add: `<citedRange unit="page">240-251</citedRange>`

File: 3001-4000/LIT3966Homily.xml [hunk 14]
```diff
@@ -53,6 +52,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <div type="bibliography">
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Tasfa1954DersanaM"/>
+                  <citedRange unit="page">164-172</citedRange>
+               </bibl>
+               <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3966Homily.xml:60]: add: `<citedRange unit="page">257-270</citedRange>`

File: 3001-4000/LIT3967Homily.xml [hunk 15]
```diff
@@ -52,32 +53,53 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                  <citedRange unit="page">181-186</citedRange>
-               </bibl>.</note>
-            <!--text to be added-->
-            <div type="textpart" subtype="incipit" xml:lang="gez">
-               <!--p. 181 -->
-               <p/>
-               <p/>
-               <!--p. 182-->
-               <p> </p>
+            <ab>በስመአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱ፡ ድርሳን፡ ዘሊቀ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ዘይትነበብ፡ በወርኅ፡
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3967Homily.xml:68]: Unclear from which manuscript this text is taken from.

File: 3001-4000/LIT3967Homily.xml [hunk 16]
```diff
@@ -52,32 +53,53 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <div type="bibliography">
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Tasfa1954DersanaM"/>
+                  <citedRange unit="page">181-186</citedRange>
+               </bibl>
+               <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3967Homily.xml:61]: add: `<citedRange unit="page">286-293</citedRange>`

File: 3001-4000/LIT3968Homily.xml [hunk 17]
```diff
@@ -79,6 +85,35 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             </div> 
                
             </div>
+         <div type="edition" xml:lang="gez">
+            <ab>በስመአብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡ ድርሳን፡ ዘቅዱስ፡<gap reason="omitted" unit="chars" quantity="3"/> ሊቀ፡ መላእክት፡ ዘይትነበብ፡ አመ፡
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3968Homily.xml:89]: Unclear from which manuscript this text was taken from.

File: 3001-4000/LIT3968Homily.xml [hunk 18]
```diff
@@ -56,6 +55,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <listRelation>
                <relation name="saws:formsPartOf" active="LIT3968Homily" passive="LIT1295Dersan"/>
             </listRelation>
+            <listBibl type="editions">
+               <bibl>
+                  <ptr target="bm:Tasfa1954DersanaM"/>
+                  <citedRange unit="page">190-201</citedRange>
+               </bibl>
+               <bibl><ptr target="bm:2006DersanaMikael"/></bibl>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3968Homily.xml:63]: add: `<citedRange unit="page">299-310</citedRange>`

File: 1001-2000/LIT1637Homily.xml [hunk 19]
```diff
@@ -15,9 +15,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             <availability>
                <licence target="http://creativecommons.org/licenses/by-sa/4.0/">
-                  <p>
+                  
                 This file is licensed under the Creative Commons
-                Attribution-ShareAlike 4.0. </p>
+                Attribution-ShareAlike 4.0. 
                </licence>
             </availability>
          </publicationStmt>
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LIT1637Homily.xml:23]: Add change note in revisionDesc

File: 3001-4000/LIT3968Homily.xml [hunk 20]
```diff
@@ -56,6 +55,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             <listRelation>
```
Comment (CarstenHoffmannMarburg) [inline 3001-4000/LIT3968Homily.xml:55]: Add change note to revisionDesc

File: 4001-5000/LIT4155MiraclesMichael.xml [hunk 21]
```diff
@@ -52,12 +52,56 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
       <body>
          <div type="bibliography">
+            <listRelation><relation name="ecrm:P129_is_about" active="LIT3959Homily" passive="PRS7049Michael"/></listRelation>
             
+            <!-- <note>???? Which miracle??? Variant of the <ref type="work" corresp="LIT3959Homily">homily</ref> by <persName ref="PRS3828Epiphani"/> on the angels and the child <persName ref="PRS11321Talafinos"/> transmitted also as <ref type="work" corresp="LIT1295Dersan#HomilyTalafinos"/>, 
+                  <ref type="work" target="LIT1295Dersan#CommemorativeTemple">Commemorative note for Sane</ref> and 
+                  <ref type="work" target="LIT1295Dersan#HomilyTalason">Homily for Hamle on Talason</ref>. </note> -->
+         </div>
+         <div type="edition">
+            <div type="textpart" xml:id="Miracle1" corresp="LIT1295Dersan#MiracleSailors">
```
Comment (CarstenHoffmannMarburg) [inline 4001-5000/LIT4155MiraclesMichael.xml:68]: The order of miracles is probably often mixed or incomplete. Why do you not select the same xml:id as in LIT1295Dersan (e.g. MiracleSailors)

File: 1001-2000/LIT1295Dersan.xml [hunk 22]
```diff
@@ -906,120 +732,73 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-                           እስከ፡ ዮም።</l>
-                        <l n="59">ወኮነ፡ ዐቢይ፡ ኀይል፡ ወመንክር፡ በእንተ፡ ስመ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ በውስተ፡ ውእቱ፡ ብሔር።</l>
-                        <l n="60">ከማሁ፡ ይኩን፡ በረከቱ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ወስእለታ፡ ለአፎምያ፡ ቅድስት፡ ላዕሌነ፡ ለዓለመ፡ ዓለም፡ አሜን።</l>
-                     </ab>
-                  </div>
+               <label xml:lang="en">
+                  <foreign xml:lang="gez">Sane</foreign>
+               </label>
+               
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilySane" corresp="LIT3964Homily">
```
Comment (CarstenHoffmannMarburg) [inline 1001-2000/LIT1295Dersan.xml:748]: Do you want to keep HomilySane here?

File: 6001-7000/LIT6631SalMikMahari.xml [hunk 23]
```diff
@@ -53,6 +54,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <div type="bibliography">
```
Comment (CarstenHoffmannMarburg) [inline 6001-7000/LIT6631SalMikMahari.xml:54]: add change note in revisionDesc

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: 3001-4000/LIT3963Homily.xml (hunk 10 above)
Comment (eu-genia) [reply 3001-4000/LIT3963Homily.xml:42]: The month was already in the title but OK

File: 4001-5000/LIT4155MiraclesMichael.xml (hunk 21 above)
Comment (eu-genia) [reply 4001-5000/LIT4155MiraclesMichael.xml:68]: When someone checks the miracles - sure. I now assume the standard order by month in the miracle collection (which is not Dersan) and only list the standard set. I however avoid the month names. Same IDs may be confusing, the link is not the same.

I have now created individual records and added corresp for all miracles here and in DM

**Comment by eu-genia** [comment]
Since manuscripts point to the references in this version I would merge it now and then I will open a new branch for all the edits. Hope this is OK.

**Pull request merged**